cloud/scope/powervs_cluster.go

```diff
@@ -2550,7 +2553,7 @@ func (s *PowerVSClusterScope) DeleteServiceInstance() (bool, error) {
 		return false, err
 	}
 	s.Info("PowerVS service instance successfully deleted")
-	return false, nil
+	return true, nil
```

Comment:
Why to re-queue in case of successful delete?

Reply:
As per the logic, we are requeuing after successful delete of a resource and wait for it to get deleted before checking again in the next reconcile to confirm if it has been removed successfully.